frontend/src/pages/ArtifactList.tsx

```diff
@@ -66,28 +73,30 @@ class ArtifactList extends Page<{}, PipelineListState> {
     const columns: Column[] = [
       {label: 'Name', flex: 1, customRenderer: this.nameCustomRenderer},
       {label: 'Version', flex: 1},
-      {label: 'Type', flex: 1},
-      {label: 'URI', flex: 1},
+      {label: 'Type', flex: 2},
+      {label: 'URI', flex: 2},
       {label: 'Workspace', flex: 1},
       {label: 'Created at', flex: 1},
     ];
 
-    const rows: Row[] = this.state.artifacts.map((a) => {
-      const type = this.artifactTypes && this.artifactTypes.get(a.type_id!) ?
-        this.artifactTypes.get(a.type_id!)!.name : a.type_id;
-      return {
-        id: a.id!,
-        otherFields: [
-          a.properties!.name.string_value,
-          a.properties!.version ? a.properties!.version!.string_value : null,
-          type,
-          a.uri,
-          a.custom_properties![CustomProperties.WORKSPACE] ?
-            a.custom_properties![CustomProperties.WORKSPACE].string_value : '',
-          formatDateString(a.properties!.create_time!.string_value),
-        ],
-      };
-    });
+    const rows: Row[] = this.state.artifacts
+      .filter((a) => !!a.properties) // We can't show much without properties
```

Comment:
Does this prevent the artifact from showing up at all in the list? That doesn't seem desirable without at least allowing users to remove the filter if they choose

Reply:
It does prevent them from displaying. I put this in place on Friday because the service was returning some items that didn't seem to be populated completely. I'm not able to get a valid response from the service at this point, but I'll retry to see if this is still the case.